libs/note-manager/src/note-manager.ts

```diff
@@ -219,7 +220,7 @@ export class NoteManager {
                 return null;
               }
             })
-            .filter((note) => Boolean(note));
+            .filter((note): note is string => Boolean(note));
```

Comment:
Explain?

Reply:
This is for the Typescript type guard on `.filter` function. Check this [post](https://www.linkedin.com/posts/mapocock_never-seen-the-is-keyword-in-typescript-activity-7125789448377626624-F4IE?utm_source=share&utm_medium=member_desktop) for more info. 

But this is the different:

| Without `note is string`  | With `note is string` |
| ------------- | ------------- |
| ![CleanShot 2023-11-08 at 14 07 45@2x](https://github.com/webb-tools/webb-dapp/assets/60747384/52661e72-9c6e-4d2e-be9e-9e7fd2258d47) | ![CleanShot 2023-11-08 at 14 07 30@2x](https://github.com/webb-tools/webb-dapp/assets/60747384/867c4d83-1573-411a-8dfe-c9d1934d90bc) |